data/firehose/firehoseBlock.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package proto;
+
+option go_package = "firehose";
+option (gogoproto.stable_marshaler_all) = true;
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+import "github.com/ElrondNetwork/elrond-go-core/data/block/block.proto";
+import "github.com/ElrondNetwork/elrond-go-core/data/transaction/log.proto";
+import "github.com/ElrondNetwork/elrond-go-core/data/transaction/transaction.proto";
+import "github.com/ElrondNetwork/elrond-go-core/data/smartContractResult/smartContractResult.proto";
+import "github.com/ElrondNetwork/elrond-go-core/data/receipt/receipt.proto";
+import "github.com/ElrondNetwork/elrond-go-core/data/rewardTx/rewardTx.proto";
+import "github.com/ElrondNetwork/elrond-go-core/data/alteredAccount/alteredAccount.proto";
+
+message FirehoseBlock {
+  bytes                   HeaderBytes = 1;
+  string                  HeaderType = 2;
+  bytes                   HeaderHash = 3;
+  Body                    Body = 4;
+  repeated AlteredAccount AlteredAccounts = 5;
+  map<string, TxWithFee>  Transactions = 6;
+  map<string, SCRWithFee> SmartContractResult = 7;
+  map<string, RewardTx>   Rewards = 8;
+  map<string, Receipt>    Receipts = 9;
+  map<string, TxWithFee>  InvalidTxs = 10;
+  repeated Log            Logs = 11;
+  bytes                   SignersIndexes = 12;
+}
+
+message FeeInfo {
```

Comment:
please use other indentations for all the structures' members (make the fields numbers on the same level). Check other files, such as `log.proto` 
_Example_
From:
```
Transaction Transaction = 1;
FeeInfo     FeeInfo = 2;
}
```

To:
```
Transaction Transaction = 1;
FeeInfo FeeInfo         = 2;
}
```